community/xxhash.hpkg

```diff
@@ -0,0 +1,29 @@
+(use ../prelude)
+(import ../core)
+
+(defsrc xxhash-src
+  :file-name
+   "xxhash-0.7.3.tar.gz"
+  :url
+   "https://github.com/Cyan4973/xxHash/archive/v0.7.3.tar.gz"
+  :hash
+   "sha256:952ebbf5b11fbf59ae5d760a562d1e9112278f244340ad7714e8556cbe54f7f7")
+
+(defpkg xxhash
+  :builder
+   (fn []
+     (os/setenv "PATH"
+                (join-pkg-paths ":" "/bin"
+                                [core/coreutils core/gcc core/make
+                                 core/sed core/which]))
```

Comment:
I tried leaving out `core/which` and it seemed to build and run ok.

Is it necessary?

Reply:
You are correct, it isn't needed.